cmd/podman/runlabel.go

```diff
@@ -23,7 +26,7 @@ var (
 		},
 		cli.BoolFlag{
 			Name:  "display",
-			Usage: "preview the command that `podman install` would execute",
+			Usage: "preview the command that `runlabel` would execute",
```

Comment:
As display is a bool type flag, maybe we should remove the "`" from the Usage, and didn't add anything behind the flag just like flag quiet.

Reply:
changed